from tools.exceptions.main_exception import MainException


class SyntaxException(MainException):
    pass
